packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -102,6 +102,49 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 			return editor.isIn('select') && editor.getSelectedShapeIds().length > 0
 		}
 
+		function scaleShapes(scaleFactor: number) {
```

Comment:
Instead of trying to implement resizing again here and missing out all our shape utils lifecycle callbacks etc, lets have this use `resizeShape` under the hood.

It should also scale shapes relative to the center of the current selection, rather from an arbitrary point on each shape.

Reply:
hah, sometimes I don't even know our own API 🙃 i looked for resize and couldn't find it for some reason.

i don't know about using the center point actually. but let's discuss more offline along w the motivation.

Reply:
updated center point - thx @SomeHats